vendor/p-timeout.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * MIT License
+
+Copyright (c) Sindre Sorhus <sindresorhus@gmail.com> (https://sindresorhus.com)
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```

Comment:
Is there a reason why are we not using npm packages and rather opt out for vendor? I don't mind it, I just am genuinely curious

Reply:
Okay I have realized this is a zinnia module, it makes sense now.